packages/modules/auth/src/services/auth-provider.ts

```diff
@@ -31,11 +31,16 @@ export default class AuthProviderService {
     try {
       return this.dependencies[`${AuthProviderRegistrationPrefix}${providerId}`]
     } catch (err) {
-      const errMessage = `
-      Unable to retrieve the auth provider with id: ${providerId}
-      Please make sure that the provider is registered in the container and it is configured correctly in your project configuration file.
-      `
+      if (err.name === "AwilixResolutionError") {
```

Comment:
NIT: Is it possible or better to do `err instanceof AwilixResolutionError`? This also works, that check is just safer typing-wise if the error object is available

Reply:
Ah yes true, I remember once this being an issue. I think this is fine, thanks JS 😄